lib/xeroizer/partner_application.rb

```diff
@@ -42,5 +42,14 @@ def initialize(consumer_key, consumer_secret, path_to_private_key, path_to_ssl_c
         end
       end
     
+    private
+
+      def read_certificate(cert)
+        if cert[-4..-1] == '.pem'
```

Comment:
This could obviously be changed, but was originally written in this form to prevent people from using the wrong file in their certificate. I probably shouldn't assume people are this bad at certificates.